open() with O_CREAT needs a mode argument (fails with _FORTIFY_SOURCE)

diff -ruN oa/fdbbackup/FileDecoder.actor.cpp ob/fdbbackup/FileDecoder.actor.cpp
--- a/fdbbackup/FileDecoder.actor.cpp	2026-09-25 17:32:33.075359057 -0700
+++ b/fdbbackup/FileDecoder.actor.cpp	2026-09-25 17:32:33.076624306 -0700
@@ -638,7 +638,7 @@
 				}
 			}
 
-			self->lfd = open(self->file.fileName.c_str(), O_WRONLY | O_CREAT | O_TRUNC);
+			self->lfd = open(self->file.fileName.c_str(), O_WRONLY | O_CREAT | O_TRUNC, 0644);
 			if (self->lfd == -1) {
 				TraceEvent(SevError, "OpenLocalFileFailed").detail("File", self->file.fileName);
 				throw platform_error();
